### PR TITLE
Add RedwoodJS icon

### PR DESCRIPTION
**Pull request by kalpeshT101**
redwoodjs

**Issue:**#3618
**Alexa rank:**[439,565](https://www.alexa.com/siteinfo/redwoodjs.com)
  

### Checklist

  - [x]  I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -5300,6 +5300,11 @@
             "hex": "764ABC",
             "source": "https://github.com/reactjs/redux/tree/master/logo"
         },
+        {
+            "title": "RedwoodJS",
+            "hex": "BC4424",
```
Comment (service-paradis) [inline _data/simple-icons.json:5305]: The color used in the SVG is `#BF4722`. Looks to be the primary color used in the website also. Where did you take this one?

File: _data/simple-icons.json [hunk 2]
```diff
@@ -5300,6 +5300,11 @@
...
+            "source": "https://redwoodjs.com/"
```
Comment (service-paradis) [inline _data/simple-icons.json:5306]: Can you change the source for https://redwoodjs.com/logos?

**Review by service-paradis — CHANGES_REQUESTED** [review]
Thanks for your contribution 😃 
The logo looks different from the original found at https://redwoodjs.com/logos
image
Can you take a look at it?

**Comment by adamrusted** [comment]
redwoodjs

Changed source, updated color and rebuilt from source - as @service-paradis' comments.

**Review by service-paradis — APPROVED** [review]
LGTM 🔴 🌳 🎉 
Thanks @kalpeshT101 and @adamrusted!

**Pull request merged**